common/membership/ringpop/test_cluster.go

```diff
@@ -146,8 +146,9 @@ func newTestCluster(
 		}).AnyTimes()
 
 	for i := 0; i < size; i++ {
+		node := i
```

Comment:
why not?
	for node := 0; node < size; node++ {

Reply:
This is the standard protection against accidental captures of the loop variable for closures and goroutines:

https://go.dev/doc/faq#closures_and_goroutines
https://github.com/golang/go/wiki/LoopvarExperiment